llama_index/finetuning/embeddings/sentence_transformer.py

```diff
@@ -35,13 +35,14 @@ def __init__(
         self.model_output_path = model_output_path
         self.model = SentenceTransformer(model_id)
 
-        # TODO: support more than 1 doc per query
+
         examples: Any = []
         for query_id, query in dataset.queries.items():
-            node_id = dataset.relevant_docs[query_id][0]
-            text = dataset.corpus[node_id]
-            example = InputExample(texts=[query, text])
-            examples.append(example)
+            for doc in dataset.relevant_docs[query_id]:
```

Comment:
if we use all relevant docs for a query, it might be slower. 
am thinking should we add option to decide whether fine tune on one query to doc pair or all query to docs pairs

Reply:
@hatianzhang I agree with your proposal. I'll try to implement it and create a new pull request.